### DISCUSSION TITLE
5/8(월) SOLID

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#130] [General]
SOLID는 정말 중요한 개념입니다. 물론 SOLID를 사용하면 코드가 예뻐지고 막 협업이 편해지고 이런 말이 있지만 적용해 본 사람으로써 개인적으로는 잘 못해서 그런가 코드가 어려워지고 현실보다는 동떨어진 코드가 나오기도 합니다. 오늘을 SOLID가 무엇인지 조사해주세요 🤔

**Comment by Eunho0922** [comment]
넵
